src/qt/bitcoin.cpp

```diff
@@ -538,14 +538,14 @@ WId BitcoinApplication::getMainWinId() const {
 static void MigrateSettings() {
     assert(!QApplication::applicationName().isEmpty());
 
-    static const QString legacyAppName("Bitcoin-Qt"),
+    static const QString legacyAppName("BitcoinABC-Qt"),
```

Comment:
Why not DeVault ?

Reply:
Because we're a BitcoinABC fork. I believe we could remove this altogether honestly, this was for migrating settings from Bitcoin-Qt to BitcoinABC-Qt